src/Components/CustomEdge.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import {
+  BaseEdge,
+  EdgeLabelRenderer,
+  getBezierPath,
+  useReactFlow,
+} from "reactflow"
+
+import "../Styles/CustomEdge.css";
+
+const onEdgeClick = (evt, id) => {
+  evt.stopPropagation();
+  alert(`remove ${id}`);
+};
+
+export default function CustomEdge(props) {
+  // if (props.source === props.target) {
+  //   return <SelfConnecting {...props} />;
+  // }
+  //   const { setEdges } = useReactFlow();
+  const [edgePath, labelX, labelY] = getBezierPath(props);
+  const [write, setWrite] = useState(false);
+  function onEdgeClick() {
+    setWrite(true);
+    console.log("fuck");
+  }
+
+  //   const onEdgeClick = () => {
```

Comment:
Please clean up the not needed comments

Reply:
sorry about those comments. I'll clean it up.

Reply:
These commented functions are still here, please clean up when you have a chance